boards/synopsys/em_starterkit/em_starterkit_emsk_em7d.dts

```diff
@@ -7,7 +7,7 @@
 /dts-v1/;
 
 #include <mem.h>
-#include <synopsys/emsk.dtsi>
+#include <synopsys/emsk-sec.dtsi>
```

Comment:
why "-sec"?

Reply:
because that is what the original board dts has?

Reply:
Fixed